macros/AddonSidebar.ejs

```diff
@@ -273,6 +273,97 @@ var text = mdn.localStringMap({
     'Blog': 'ブログ',
     'Forums': 'フォーラム',
     '#Contact_us': 'コンタクト'
+  },
+  'ko': {
+    'WebExtensions': '브라우저 확장 기능',
+    'WebExtensions#Getting_started': '시작하기',
+    'WebExtensions/What_are_WebExtensions': '확장 기능이 무엇인가요?',
+    'WebExtensions/Your_first_WebExtension': '첫 번째 확장 기능',
+    'WebExtensions/Your_second_WebExtension': '두 번째 확장 기능',
+    'WebExtensions/Anatomy_of_a_WebExtension': '확장 기능의 구조',
+    'WebExtensions/Examples': '확장 기능 예시',
+    'WebExtensions/What_next_': '다음은 무엇인가요?',
+    'WebExtensions#Concepts': '개념',
+    'WebExtensions/Using_the_JavaScript_APIs': 'JavaScript API 사용하기',
+    'WebExtensions/Content_scripts': '컨텐츠 스크립트',
+    'WebExtensions/Match_patterns': '일치 패턴',
+    'WebExtensions/Working_with_files': '파일 작업',
+    'WebExtensions/Internationalization': '국제화',
+    'WebExtensions/Security_best_practices': '보안 모범 사례',
+    'WebExtensions/Content_Security_Policy': '컨텐츠 보안 정책',
+    'WebExtensions/Native_messaging': '네이티브 메시징',
+    'WebExtensions#User_Interface': '사용자 인터페이스',
+    'WebExtensions/user_interface': '사용자 인터페이스',
+    'WebExtensions/user_interface/Browser_action': '도구 모음 버튼',
+    'WebExtensions/user_interface/Page_actions': '주소 표시줄 버튼',
+    'WebExtensions/user_interface/Sidebars': '사이드바',
+    'WebExtensions/user_interface/Context_menu_items': '컨텍스트 메뉴 항목',
+    'WebExtensions/user_interface/Options_pages': '옵션 페이지',
+    'WebExtensions/user_interface/Extension_pages': '확장 기능 페이지',
+    'WebExtensions/user_interface/Notifications': '알림',
+    'WebExtensions/user_interface/Omnibox': '주소 표시줄 제안',
+    'WebExtensions/user_interface/devtools_panels': '개발자 도구 패널',
+    'WebExtensions#How_to': '방법',
+    'WebExtensions/Intercept_HTTP_requests': 'HTTP 요청 가로채기',
+    'WebExtensions/Modify_a_web_page': '웹 페이지 수정하기',
+    'WebExtensions/Safely_inserting_external_content_into_a_page': '외부 컨텐츠 삽입하기',
+    'WebExtensions/Add_a_button_to_the_toolbar': '도구 모음에 버튼 추가하기',
+    'WebExtensions/Implement_a_settings_page': '설정 페이지 구현하기',
+    'WebExtensions/Working_with_the_Tabs_API': 'Tabs API 사용하기',
+    'WebExtensions/Work_with_the_Bookmarks_API': 'Bookmarks API 사용하기',
+    'WebExtensions/Work_with_the_Cookies_API': 'Cookies API 사용하기',
+    'WebExtensions/Work_with_contextual_identities': 'contextual identities 사용하기',
+    'WebExtensions/onboarding_upboarding_offboarding_best_practices': '사용자의 온보드, 업보드 및 오프보드',
+    'WebExtensions#Porting': '포팅',
+    'WebExtensions/Firefox_differentiators': 'Firefox 의 차별화',
```

Comment:
```suggestion
    'WebExtensions/Firefox_differentiators': 'Firefox의 차별화',
```
It's my personal opinion, but I prefer not to add a space.

Same for :322, :330